### PR TITLE
[zh-cn] sync resource-quotas configure-redis-using-configmap tutorials/security/* hello-minikube

### PR DESCRIPTION
**Pull request by my-git9**
```
# Changes to be committed:
#       modified:   content/zh-cn/docs/concepts/policy/resource-quotas.md
#       modified:   content/zh-cn/docs/tutorials/configuration/configure-redis-using-configmap.md
#       modified:   content/zh-cn/docs/tutorials/hello-minikube.md
#       modified:   content/zh-cn/docs/tutorials/security/apparmor.md
#       modified:   content/zh-cn/docs/tutorials/security/cluster-level-pss.md
#       modified:   content/zh-cn/docs/tutorials/security/seccomp.md

```

### CODE REVIEW

File: content/zh-cn/docs/tutorials/security/apparmor.md [hunk 1]
```diff
@@ -329,7 +329,7 @@ Next, we'll run a simple "Hello AppArmor" pod with the deny-write profile:
 -->
 接下来，我们将运行一个带有拒绝写入配置文件的简单 “Hello AppArmor” Pod：
 
-{{< codenew file="pods/security/hello-apparmor.yaml" >}}
+{{<%code_sample file="pods/security/hello-apparmor.yaml" %}}
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/security/apparmor.md:332]:
```suggestion
{{<% code_sample file="pods/security/hello-apparmor.yaml" %}}
```

**Comment by tengqm** [comment]
/approve

Not sure the change to `content/zh-cn/docs/tutorials/security/apparmor.md` is correct regarding the `**` space thing.

**Comment by my-git9** [comment]
> /approve
> 
> Not sure the change to `content/zh-cn/docs/tutorials/security/apparmor.md` is correct regarding the `**` space thing.

Seems no problems：https://deploy-preview-43729--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tutorials/security/apparmor/

**Pull request merged**